src/components/search/generic-topic.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {FunctionComponent} from 'react'
+import Markdown from 'react-markdown'
+import Image from 'next/image'
+import {NextSeo} from 'next-seo'
+
+type TopicProps = {
+  title: string
+  children?: string
+  imageUrl?: string
+  className?: string
+  description?: string
+}
+
+const GenericTopic: FunctionComponent<TopicProps> = ({
```

Comment:
I cut and pasted `Topic` component for this with some specific changes like seo tags etc